v1/contracts/Withdrawable.sol

```diff
@@ -0,0 +1,22 @@
+/*
+    Contract that allows the owner to withdraw any ETH and/or ERC20 tokens that the contract holds.
+*/
+
+pragma solidity ^0.5.0;
+
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+
+contract Withdrawable is Ownable {
```

Comment:
This should be in `common/contracts`, no?

Although if @ptare already started working on moving `v0` to its own branch, doesn't really matter.